scripts/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,538 @@
+// FIXME sloc publish errors
```

Comment:
better to start with some description what this is and not with a TODO list :smile: